pages/common/sdiff.md

```diff
@@ -0,0 +1,23 @@
+# sdiff
+
+> Compare the differences and/or merge between two files.
+
+- Compare 2 files:
+
+`sdiff {{path/to/first_file}} {{path/to/second_file}}`
+
+- Compare 2 files, ignoring all tabs and whitespace:
+
+`sdiff -W {{path/to/first_file}} {{path/to/second_file}}`
+
+- Compare 2 files, ignoring whitespace at the end of lines:
+
+`sdiff -Z {{path/to/first_file}} {{path/to/second_file}}`
+
+- Case insensitively compare 2 files:
+
+`sdiff -i {{path/to/first_file}} {{path/to/second_file}}`
+
+- Compare 2 files and then merge, creating a new file:
+
+`sdiff -o {{path/to/new_merge_file}} {{path/to/first_file}} {{path/to/second_file}}`
```

Comment:
path/to/merged_file

Reply:
I also quite not sure on this, 'new' will communicate clearly to the reader that this is a new file that create after merging.

What do you think? @agnivade

Reply:
It must always be a new file which gets created right. By definition, if you "create" something, that is new. So I don't think there is any ambiguity here.

Reply:
Agree, I'll change it.